frontend/src/metabase/admin/settings/components/ApiKeys/CreateApiKeyModal.tsx

```diff
@@ -21,9 +22,14 @@ export const CreateApiKeyModal = ({ onClose }: { onClose: () => void }) => {
   const secretKey = response?.data?.unmasked_key || "";
 
   const handleSubmit = useCallback(
-    async vals => {
-      await createApiKey(vals);
-      setModal("secretKey");
+    async (vals: { group_id: number | null; name: string }) => {
+      // bad
```

Comment:
idk what to say about this comment... 🤨

Reply:
Haha, I used that as markers for the `any` keys. I must have forgot to remove the comment 😂